tests/net/lib/mqtt_pubsub/src/test_mqtt_pubsub.c

```diff
@@ -101,8 +101,7 @@ void publish_handler(struct mqtt_client *const client,
 		    const struct mqtt_evt *evt)
 {
 	int rc;
-	u8_t buf[16];
-	u32_t offset = 0U;
+	u8_t buf[2048];
```

Comment:
If you make this static, then we can use the default stack size. In this unit test, there is probably only one caller of this function anyway.

Reply:
Nice, thanks!